package.json

```diff
@@ -69,8 +69,10 @@
     "compare-versions": "^3.6.0",
     "depcheck": "^1.0.0",
     "eslint": "^7.5.0",
+    "eslint-config-prettier": "^7.1.0",
```

Comment:
Question for the lerna pros. Should the new package not be included in the list of workspaces in the root package.json

Reply:
I think it should, so that we always use the local state